src/main/java/com/bzabek/ticketbookingapp/entities/Room.java

```diff
@@ -0,0 +1,49 @@
+package com.bzabek.ticketbookingapp.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+public class Room {
+
+    @Id
+    private Long id;
+    @OneToMany(mappedBy = "room")
+    @JsonIgnore
```

Comment:
jaka jest wada użycia tego samego obiektu jako DTO w HTTP i encji bazodanowej?

Reply:
Ten sam obiekt jest używany w warstwie biznesowej, danych i prezentacji. Na dłuższą mete może to utrudnić pracę.
Używanie DTO pozwala zoptymalizować sposób dostępu do BD. 
Jeżeli przy wysyłaniu odpowiedzi uzyjemy encji może wystapić nieskończona pętla związana z tym, że encje(tabele) są ze sobą nawzajem powiązane, użycie DTO może temu zapobiec.
DTO może mieć mniej pól niż encja, np. Id jest gnerowane automatycznie i nie przekazujemy tej wartości do BD.